examples/simple_example.rs

```diff
@@ -41,14 +41,35 @@ extern crate libc;
 use traceme::Tracer;
 use libc::getpid;
 
+use traceme::backends::DummyTracer;
+#[cfg(perf_pt)]
+use traceme::backends::PerfPTTracer;
+
```

Comment:
Double blank line.

Reply:
This was intentional to separate imports from functions. Nevertheless, if thats the style we prefer, ok.